### PR TITLE
Colw/graphql validators

### PR DESCRIPTION
**Pull request by colw**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -295,32 +271,18 @@ export default {
...
     },
-    statusColor() {
-      if (this.validator.jailed) return `red`
-
-      if (parseFloat(this.validator.status) === 0) return `orange`
-
-      return `green`
-    },
-    // empty descriptions have a strange '[do-not-modify]' value which we don't want to show
     website() {
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue]: We don't want to show `[do-not-modify]` which is/was a default value and shows in some cases

Comment (colw) [reply src/components/staking/PageValidator.vue]: I caught this with the noBlanks filter (not the best name)

but… I don't think it works well for website. I'll fix this.

File: src/components/common/TmPage.vue [hunk 2]
```diff
@@ -77,7 +77,7 @@ export default {
       default: false
     },
     error: {
-      type: Error,
+      type: [Error, Boolean],
```
Comment (faboweb) [inline src/components/common/TmPage.vue]: We have a rule about not allowing multiple types per property

File: src/components/staking/PageValidator.vue [hunk 3]
```diff
@@ -503,17 +472,17 @@ span {
   border-radius: 0.25rem;
 }
 
-.validator-status.red {
+.validator-status.jailed {
```
Comment (faboweb) [inline src/components/staking/PageValidator.vue]: nice semantic classes

File: src/components/common/TmPage.vue (hunk 2 above)
Comment (colw) [reply src/components/common/TmPage.vue]: Good call. `TMPage` already treated it like a Boolean, so I've removed the `Error` type. Other values will be coerced to Boolean.

**Pull request merged**